subtests/docker_cli/run_volumes/run_volumes.py

```diff
@@ -42,7 +42,12 @@
 
 
 class run_volumes(SubSubtestCaller):
-    config_section = 'docker_cli/run_volumes'
+
+    def run_once(self):
+        ptc = self.config['pretest_cmd'].strip()
+        if ptc != '' and ptc is not None:
+            utils.run(ptc, timeout=self.config['docker_timeout'])
+        super(run_volumes, self).run_once()
```

Comment:
run_once() in this case actually executes all the sub-subtests, so it has to come last.  Probably I should put the ptc thing into initialize() instead, okay I'll fix that, thx.